types/koa-bodyparser/index.d.ts

```diff
@@ -21,8 +21,8 @@ import * as Koa from "koa";
 
 declare module "koa" {
     interface Request {
-        // any declaration breaks types intellisense and type safety, keep it at least Record<string, unknown>
-        body?: Record<string, unknown>;
+        // Fix #62921
+        body?: any;
```

Comment:
I think this should have been like this instead, using `unknown` in place of `any`:

```
body?: unknown;
```

Because, `any` doesn't allow us to use intersection to provide more specific types for `body` when needed:

<img width="428" alt="Screenshot 2022-11-04 at 4 11 45 pm" src="https://user-images.githubusercontent.com/1754873/199894834-cc7e63b6-58c6-4167-a5ef-924c55814acf.png">

Compared to:

<img width="434" alt="Screenshot 2022-11-04 at 4 12 08 pm" src="https://user-images.githubusercontent.com/1754873/199894921-de073fca-1f41-4634-8cbc-0bb528aaa5de.png">

Reply:
> I think this should have been like this instead, using `unknown` in place of `any`:
> 
> ```
> body?: unknown;
> ```
> 
> Because, `any` doesn't allow us to use intersection to provide more specific types for `body` when needed:
> 
> <img alt="Screenshot 2022-11-04 at 4 11 45 pm" width="428" src="https://user-images.githubusercontent.com/1754873/199894834-cc7e63b6-58c6-4167-a5ef-924c55814acf.png">
> 
> Compared to:
> 
> <img alt="Screenshot 2022-11-04 at 4 12 08 pm" width="434" src="https://user-images.githubusercontent.com/1754873/199894921-de073fca-1f41-4634-8cbc-0bb528aaa5de.png">

Oh man I think you're right. 😱

Reply:
Here is a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63074 .

Reply:
Good work figuring this out. This problem with the typing of the body parameter has been ongoing for years at this point, so hopefully this will resolve it more or less permanently?